Sources/API/Routes/SubscriptionRoutes.swift

```diff
@@ -60,17 +60,22 @@ public final class SubscriptionRoutes {
         if let plan = plan {
             body["plan"] = Node(plan)
         }
+        
         if let appFeePercent = applicationFeePercent {
             body["application_fee_percent"] = Node(appFeePercent)
         }
+        
         if let couponId = couponId {
             body["coupon"] = Node(couponId)
         }
-        if let items = items?.object {
-            for (key,value) in items {
-                body["items[\(key)"] = value
+        
+        if let items = items?.array {
+            
+            for x in 0..<items.count {
```

Comment:
Could you add the enumerated version here

Reply:
Done